week1/homework/test/routes.spec.js

```diff
@@ -54,8 +54,11 @@ describe("GET /meal", () => {
     }
     const response = await request(app).get("/meal");
     expect(response.statusCode).toBe(200);
-    expect(typeof response.body).toBe("object");
     expect(Array.isArray(response.body)).toBeFalsy();
+    expect(typeof response.body).toBe("object");
+    expect(response.body).toEqual(
+      expect.objectContaining({ id: expect.any(Number) })
+    );
   });
 });
 
```

Comment:
Good!
Now the test fails 👌🏽 

![image](https://user-images.githubusercontent.com/64137994/155818753-0e193370-aa90-4fb3-aa81-dffb3a2df353.png)


Same could be imlemented for "/reservation" (line 69)